plugins/modules/file_servers/backup.py

```diff
@@ -116,7 +117,11 @@ def main():
         agent = client.agents.get('File System')
         backupset = agent.backupsets.get(agent.backupsets.default_backup_set if not module.params.get('backupset') else module.params.get('backupset'))
         subclient = backupset.subclients.get(backupset.subclients.default_subclient if not module.params.get('subclient') else module.params.get('subclient'))
-        backup = subclient.backup()
+        backup_level = module.params.get('backup_level')
```

Comment:
can we do something with a ternary operator?
something like backup_level = module.params.get('backup_level') if module.params.get('backup_level') else "incremental"
and then always pass the backup_level to the backup call

Reply:
actually it would be better to use the dict's get default option instead
we can do something like backup_level = module.params.get('backup_level', 'incremental') instead